spp_event_data/models/event_data.py

```diff
@@ -0,0 +1,49 @@
+# Part of OpenSPP. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo import _, api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class OpenSPPEventData(models.Model):
```

Comment:
Missing the reference to the Registrant

Reply:
Fixed with [commit](https://github.com/OpenSPP/openspp-registry/pull/52/commits/01cb90811f52afc2ecb0251d3f4415bd316f9fb9)